### ISSUE TITLE
Simulation wizard: Disable "substances" if waterquality not in scope of organisation's contract

### ISSUE THREAD

**Issue by leendertvanwolfswinkel** [nens/threedi-api-qgis-client#625]
Show the following tooltip:

"Your organisation's contract does not include water quality simulations. Please get in touch to be informed about the possibilities."

**Comment by leendertvanwolfswinkel** [comment]
Tested, works perfectly

Please do NOT merge yet until we have everything configured on prod

**Comment by leendertvanwolfswinkel** [comment]
@benvanbasten-ns this can now also be merged
